app/helpers/template_helpers.py

```diff
@@ -1,304 +1,376 @@
+    mobile_logo: str = "nisra-logo-en-mobile"
+    copyright_declaration: LazyString = lazy_gettext(
+        "Crown copyright and database rights 2021 NIMA MOU577.501."
+    )
+    copyright_text: LazyString = lazy_gettext(
+        "Use of address data is subject to the terms and conditions."
+    )
+    footer_links: Iterable[MutableMapping] = field(
+        default_factory=lambda: [
+            Link(
+                lazy_gettext("Help"),
+                f"{CENSUS_NIR_BASE_URL}/help/help-with-the-questions/online-questions-help/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Contact us"), f"{CENSUS_NIR_BASE_URL}/contact-us/"
+            ).__dict__,
+        ],
+        compare=False,
+        hash=False,
+    )
+    footer_legal_links: Iterable[Mapping] = field(
+        default_factory=lambda: [
+            Link(lazy_gettext("Cookies"), f"{CENSUS_NIR_BASE_URL}/cookies/").__dict__,
+            Link(
+                lazy_gettext("Accessibility statement"),
+                f"{CENSUS_NIR_BASE_URL}/accessibility-statement/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Privacy and data protection"),
+                f"{CENSUS_NIR_BASE_URL}/privacy-and-data-protection/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Terms and conditions"),
+                f"{CENSUS_NIR_BASE_URL}/terms-and-conditions/",
+            ).__dict__,
+        ],
+        compare=False,
+        hash=False,
+    )
+    powered_by_logo: str = "nisra-logo-black-en"
+    powered_by_logo_alt: str = "NISRA - Northern Ireland Statistics and Research Agency"
+    account_service_url = CENSUS_NIR_BASE_URL
+    data_layer: Iterable[Mapping] = field(
+        default_factory=lambda: [{"nisra": True}], compare=False
+    )
 
-    Replaces variable with ellipsis and strips any HTML tags.
 
-    :param (str) content: Input string.
-    :returns (str): Modified string.
-    """
-    if content is not None:
-        # Replace piping with ellipsis
-        content = re.sub(r"{.*?}", "…", content)
-        # Strip HTML Tags
-        content = re.sub(r"</?[^>]+>", "", content)
-    return content
+def generate_context(
+    theme: str,
+    language: str,
+    is_post_submission: bool,
+    include_csrf_token: bool,
+) -> dict[str, Any]:
+    return ContextHelper(
+        theme,
+        language,
+        is_post_submission,
+        include_csrf_token,
+        context_options(theme, language),
+    ).context
+
+
+@lru_cache
+def context_options(theme: str, language: str) -> SurveyConfig:
+    context_options_mapping = {
+        "business": SurveyConfig,
+        "health": SurveyConfig,
+        "social": SurveyConfig,
+        "census": (
+            CymraegCensusSurveyConfig if language == "cy" else CensusSurveyConfig
+        ),
+        "default": SurveyConfig,
+        "census-nisra": CensusNISRASurveyConfig,
+    }
+    return context_options_mapping[theme]()
```

Comment:
Now that `ContextOptions` has been renamed `SurveyConfig, the naming needs be looked at throughout this file e.g. the function name and variables here should be survey config rather than context options.

Reply:
Done